ci-operator/step-registry/hypershift/hive/workflow/hypershift-hive-workflow-workflow.yaml

```diff
@@ -0,0 +1,26 @@
+workflow:
+  as: hypershift-hive-workflow
+  documentation: |-
+    The HyperShift hive workflow executes tests against a new ephemeral
+    HyperShift cluster. Administrative access to the control plane is provided
+    via the `KUBECONFIG` environment variable.
+
+    Note that tests will execute once the control plane is ready, regardless of
+    whether nodes have been requested, have joined, or are available.
+
+    The control plane is hosted on hive cluster.
+
+    OpenShift versions supported by this workflow are 4.12.x.
+    OpenShift 4.13.x are not supported and will cause hypershift-aws-create to stall
+
+  steps:
+    post:
+    - chain: hypershift-dump
+    - chain: hypershift-hive-destroy
+    pre:
+    - ref: openshift-cluster-bot-rbac
+    - chain: hypershift-hive-credentials
+    - chain: hypershift-hive-create
```

Comment:
These files don't need to be in alphabetical order, so you can use the more natural `pre`/`post` order.